src/main/java/org/openhab/tools/analysis/checkstyle/RequireBundleCheck.java

```diff
@@ -31,6 +32,10 @@
 public class RequireBundleCheck extends AbstractStaticCheck {
     private final Logger logger = LoggerFactory.getLogger(RequireBundleCheck.class);
 
+    private final static String REQUIRE_BUNDLE_HEADER_NAME = "Require-Bundle";
+    private final static String FRAGMENT_HOST_HEADER_NAME = "Fragment-Host";
+    private final static String BUNDLE_SYMBOLIC_NAME_HEADER_NAME = "Bundle-SymbolicName";
```

Comment:
Could you move these constants to the `org.openhab.tools.analysis.checkstyle.api.CheckConstants` so they can be reused in other checks ?